src/utils/audio.js

```diff
@@ -665,3 +667,78 @@ export function window_function(window_length, name, {
 
     return window;
 }
+
+export class RawAudio {
+
+    /**
+     * Create a new `RawAudio` object.
+     * @param {Float32Array} audio
+     * @param {number} sampling_rate
+     */
+    constructor(audio, sampling_rate) {
+        this.audio = audio;
+        this.sampling_rate = sampling_rate;
+    }
+
+    /**
+     * Save the audio to a wav file.
+     * Wav file specs : https://fr.wikipedia.org/wiki/Waveform_Audio_File_Format
+     * @param {string} path
+     * @returns {any} if webworker return Uint8Array, else none
+     */
+    save(path) {
+        let audio, sampling_rate, out, wav_header, buf_size, nums
+
+        ({
+            audio,
+            sampling_rate
+        } = this)
+        buf_size = audio.buffer.byteLength
```

Comment:
This suggests a maximum buffer size? Could we throw an error if this is larger than allowed?